app/scripts/modules/core/src/widgets/cluster/ClusterMatches.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { IClusterMatch } from './clusterMatches.component';
+import { NgReact } from 'core/reactShims/ngReact';
+
+export interface IClusterMatchesProps {
+  matches: IClusterMatch[]
+}
+
+export class ClusterMatches extends React.Component<IClusterMatchesProps> {
+
+  public render() {
+    const { matches } = this.props;
+    const { AccountTag } = NgReact;
+    if (!matches || !matches.length) {
+      return (<div>(no matches)</div>);
+    }
+    return (
+      <div>
+        {matches.map((match: IClusterMatch, index: number) => (
+          <div key={index}>
+            <AccountTag account={match.account}/>
+            <span className="break-word">{match.name}</span>
+            {match.regions && match.regions.length && <i> in {match.regions.join(', ')}</i>}
```

Comment:
Do we use i for italics and icons?

Reply:
We don't do this consistently. You'll see a lot of `<i>`s and `<em>`s throughout the codebase.

(I never connected `i` with `icon` until you mentioned it above, but it explains why font awesome uses it)